### PR TITLE
Fix incorrect link in zh-CN and kr README.md

### PR DESCRIPTION
**Pull request by win5923**
Fix incorrect link and add zh_CN translations for kubernetes setup.

### CODE REVIEW

File: k8s/readme.md
Comment (thijsvanloef) [inline k8s/readme.md:1]: Not a fan of duplicating the content to translate a single sentence

**Review by thijsvanloef — CHANGES_REQUESTED** [review]

File: k8s/readme.md
Comment (win5923) [reply k8s/readme.md:1]: You mean to split it into two commits?

Comment (thijsvanloef) [reply k8s/readme.md:1]: only remove the zh-CN/kr from the k8s/readme.md file, after that, ready to merge

Comment (win5923) [reply k8s/readme.md:1]: Done!

**Review by thijsvanloef — APPROVED** [review]

**Pull request merged**